README.md

```diff
@@ -24,7 +24,7 @@ To run Umbrel on:
 ### Instructions:
 
 1. Make sure you've everything ready:
-    1. [Raspberry Pi 4 with 4GB or 8GB RAM](https://www.amazon.com/gp/product/B08C4SK5C3/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B08C4SK5C3&linkCode=as2&tag=umbrel08-20&linkId=d3f17fd4813568c4863d239839311d67)
+    1. [Raspberry Pi 4 with 4GB or 8GB RAM](https://www.amazon.com/CanaKit-Raspberry-Basic-Kit-8GB/dp/B08DJ9MLHV/ref=sr_1_3?crid=2QCDYXW7U4JXD&dchild=1&keywords=raspberry+pi+4+8gb&qid=1615611754&sprefix=rasp%2Caps%2C309&sr=8-3)
```

Comment:
```suggestion
    1. [Raspberry Pi 4 with 4GB or 8GB RAM](https://www.amazon.com/gp/product/B07TC2BK1X/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B07TC2BK1X&linkCode=as2&tag=umbrel08-20&linkId=551cb394f0206b2c3e87b68181a05697)
```

This is the link from getumbrel.com, buying a kit is just a waste of money, because this kit for example contains a power supply that doesn't work, so users would need to buy a second power supply when they already paid for one in the kit.